docs/user/Build.md

```diff
@@ -3,9 +3,9 @@
 ## Install dependencies
 
 To install dependencies to develop or build and run OpenROAD locally,
-follow [this procedure](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html).
+follow [this procedure](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html#clone-and-install-dependencies).
```

Comment:
We should re-write this with the correct instructions on how to run the installer.

Reply:
@vvbandeira I just added some instructions, let me know if that works